[2024-06-18T11:03:56.521+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-18T11:03:56.540+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag.second_task scheduled__2021-10-27T00:00:00+00:00 [queued]>
[2024-06-18T11:03:56.549+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag.second_task scheduled__2021-10-27T00:00:00+00:00 [queued]>
[2024-06-18T11:03:56.549+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-18T11:03:56.570+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): second_task> on 2021-10-27 00:00:00+00:00
[2024-06-18T11:03:56.578+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3111) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-18T11:03:56.580+0000] {standard_task_runner.py:63} INFO - Started process 3129 to run task
[2024-06-18T11:03:56.580+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 'second_task', 'scheduled__2021-10-27T00:00:00+00:00', '--job-id', '285', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmp3ejx613h']
[2024-06-18T11:03:56.582+0000] {standard_task_runner.py:91} INFO - Job 285: Subtask second_task
[2024-06-18T11:03:56.631+0000] {task_command.py:426} INFO - Running <TaskInstance: test_dag.second_task scheduled__2021-10-27T00:00:00+00:00 [running]> on host 195daacf7793
[2024-06-18T11:03:56.726+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_dag' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2021-10-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-10-27T00:00:00+00:00'
[2024-06-18T11:03:56.727+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-18T11:03:56.737+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-18T11:03:56.738+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2024-06-18T11:03:56.750+0000] {subprocess.py:86} INFO - Output:
[2024-06-18T11:03:56.751+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1!
[2024-06-18T11:03:56.752+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-18T11:03:56.753+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-18T11:03:56.779+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=second_task, run_id=scheduled__2021-10-27T00:00:00+00:00, execution_date=20211027T000000, start_date=20240618T110356, end_date=20240618T110356
[2024-06-18T11:03:56.835+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-18T11:03:56.844+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
